lib/spack/spack/package.py

```diff
@@ -483,37 +483,67 @@ class SomePackage(Package):
     #
     # These are default values for instance variables.
     #
-    """By default we build in parallel.  Subclasses can override this."""
     parallel = True
+    """By default we build in parallel.  Subclasses can override this."""
```

Comment:
According to http://www.sphinx-doc.org/en/stable/ext/autodoc.html#directive-autoattribute:

> For module data members and class attributes, documentation can either be put into a comment with special formatting (using a `#:` to start the comment instead of just `#`), or in a docstring _after_ the definition.

You can see that all of these docstrings were one-off in the documentation: http://spack.readthedocs.io/en/latest/spack.html#spack.package.PackageBase.use_xcode